ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.js

```diff
@@ -101,12 +102,13 @@ function ClustersTablePanel({ selectedClusterId, setSelectedClusterId, searchOpt
         </div>
     ));
 
+    const { version } = metadata;
+
     const installMenuOptions = [
         <DropdownItem
             key="link"
-            isExternalLink
             description="Cluster installation guides"
-            href="https://docs.openshift.com/acs/installing/acs-installation-platforms.html"
+            href={getVersionedDocs(version, 'installing/acs-installation-platforms.html')}
```

Comment:
Without conditional rendering to include item only if `version` is non-empty, `href` has possible value:
https://docs.openshift.com/acs//installing/acs-installation-platforms.html

By design, the function does not support fallback to unversioned docs link, because it is not safe to assume/require that links to specific pages for older versions like 3.74 will exist in docs for latest version (for example, in case of subsequent reorganization).